src/python/WMComponent/TaskArchiver/CleanCouchPoller.py

```diff
@@ -94,6 +94,7 @@ def __init__(self, config):
         self.perfDashBoardMaxLumi = getattr(config.TaskArchiver, "perfDashBoardMaxLumi", 9000)
         self.dashBoardUrl = getattr(config.TaskArchiver, "dashBoardUrl", None)
         self.DataKeepDays = getattr(config.TaskArchiver, "DataKeepDays", 0.125)  # 3 hours
+        self.cleanCouchDelayHours = getattr(config.TaskArchiver "cleanCouchDelayHours", 24)
```

Comment:
There is a small typo here:
```suggestion
        self.cleanCouchDelayHours = getattr(config.TaskArchiver, "cleanCouchDelayHours", 24)
```

Reply:
thanks for catching this @germanfgv .

I am fixing it immediately

Reply:
done